sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/Generated/Models/Container.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ContainerInstance.Models
+{
+    /// <summary> A container instance. </summary>
+    public partial class Container
```

Comment:
Single word class name, please rename it.

Reply:
Renamed with ContainerInstanceContainer